var/spack/repos/builtin/packages/esmf/package.py

```diff
@@ -26,6 +26,7 @@ class Esmf(MakefilePackage):
     # Develop is a special name for spack and is always considered the newest version
     version("develop", branch="develop")
     # generate chksum with spack checksum esmf@x.y.z
+    version("8.4.1", sha256="1b54cee91aacaa9df400bd284614cbb0257e175f6f3ec9977a2d991ed8aa1af6")
```

Comment:
We already have that in spack-stack, so we'll need to fix the merge conflicts when this comes down to our fork